.husky.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We should remove this file. In v7 there should be used the separate hooks instead of this config file